.github/CODEOWNERS

```diff
@@ -23,6 +23,10 @@
 /modules/light-clients/ @colin-axner @AdityaSripal @damiannolan
 /proto/ibc/lightclients/ @colin-axner @AdityaSripal @damiannolan
 
+# CODEOWNERS for 08-wasm light client module
+
+/modules/light-clients/08-wasm/ @colin-axner @AdityaSripal @damiannolan @charleenfei @chatton @DimitrisJim @srdtrk
```

Comment:
you can add yourself :)

Reply:
Thank you, but I don't think I deserve the honour of being part of such an exclusive club. ;)